modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePreloader.java

```diff
@@ -85,10 +85,11 @@
      * @param assignments Assignments to add.
      * @param forcePreload {@code True} if preload requested by {@link ForceRebalanceExchangeTask}.
      * @param rebalanceId Rebalance id created by exchange thread.
-     * @param next Runnable responsible for cache rebalancing chain.
+     * @param next Rebalance's future follows after the current one.
```

Comment:
Here must be the same comment as in demander method:
A next rebalance routine in chain.